src/containers/NFT/NFTHeader/index.js

```diff
@@ -1,26 +1,48 @@
-import React, { Component, useEffect, useContext, useState } from 'react'
-import { useTranslation, withTranslation } from 'react-i18next'
+import React, { useEffect, useContext, useState } from 'react'
+import { useTranslation } from 'react-i18next'
 import { Link } from 'react-router-dom'
 import PropTypes from 'prop-types'
-import { connect } from 'react-redux'
-import { bindActionCreators } from 'redux'
-import { loadNFTState } from './actions'
+import { useQuery } from 'react-query'
```

Comment:
this file is too big. there are really multiple components in here. it should be broken up into multiple files